src/Raven.Server/Config/Settings/FlagSetting.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Raven.Server.Config.Settings
+{
+    public struct FlagSetting<T> where T : struct, IConvertible
```

Comment:
Why aren't we just using an enum flag option?
Instead of `None | Local` it is `None, Local`